doc/nrf/includes/boardname_tables/sample_boardnames.txt

```diff
@@ -239,3 +239,15 @@ Set used by Tests (Crypto)
 +--------------------------------+-----------+------------------------------------------------+--------------------------------+
 
 .. set16_end
+
+Set used by samples (USB-UART bridge, Connectivity bridge)
```

Comment:
should USB-UART bridge be removed ? @jorgenmk

Reply:
It is deprecated, but still present in the samples directory. So the text as it stands is correct.